native/src/routePlannerFrontEnd.cpp

```diff
@@ -147,6 +147,30 @@ bool RoutePlannerFrontEnd::hasSegment(vector<SHARED_PTR<RouteSegmentResult>>& re
 	return false;
 }
 
+vector<SHARED_PTR<RouteSegmentResult>> RoutePlannerFrontEnd::searchRouteByPoints(RoutingContext* ctx,
```

Comment:
Need to fix searchRouteInternalPrepare and do not create new method!

Reply:
refactored